--- ./etc/macports/macports.conf.default	2013-11-18 10:53:23.000000000 +0000
+++ ./etc/macports/macports.conf	2013-11-18 11:03:44.000000000 +0000
@@ -110,6 +110,7 @@
 # https://trac.macports.org/wiki/Mirrors#MacPortsSource for a list of
 # available servers.
 #rsync_server        	rsync.macports.org
+rsync_server            macports.internal.sanger.ac.uk
 
 # Location of MacPorts base sources on rsync_server. If this references
 # a .tar file, a signed .rmd160 file must exist in the same directory
@@ -197,3 +198,11 @@
 # removing unnecessary files and directories prior to pkg or mpkg
 # deployment.
 #pkg_post_unarchive_deletions	include share/doc share/man
+
+# Target MacOS X 10.6 (Snow Leopard) and above
+#
+macosx_deployment_target	  10.6
+
+# TEMP https://trac.macports.org/ticket/39850
+#
+sandbox_enable no
--- ./etc/macports/sources.conf.default	2013-01-14 10:28:16.000000000 +0000
+++ ./etc/macports/sources.conf	2013-01-14 10:41:07.000000000 +0000
@@ -28,4 +28,6 @@
 # If an rsync URL points to a .tar file, a signed .rmd160 must exist next to
 # it on the server and will be used to verify its integrity.
 
-rsync://rsync.macports.org/release/tarballs/ports.tar [default]
+file://OTT_REL_MACOS_LOCAL_PORTS
+#rsync://rsync.macports.org/release/tarballs/ports.tar [default]
+rsync://macports.internal.sanger.ac.uk/release/ports/ [default]
